index.html

```diff
@@ -171,6 +171,7 @@ <h3>Sample Apps</h3>
         <li><a href="https://github.com/foam-framework/foam/wiki/MBug">Mobile QuickBug (MBug)</a> <a href="https://github.com/foam-framework/foam/tree/master/apps/mbug/">(source)</a></li>
         <li><a href="apps/gmail/main.html">GMail Demo</a> <a href="https://github.com/foam-framework/foam/tree/master/js/com/google/mail">(source)</a></li>
         <li><a href="https://chrome.google.com/webstore/detail/chrome-app-builder/ighkikkfkalojiibipjigpccggljgdff">Chrome App Builder</a></li>
+        <li>Java profiling generously enabled by <a href="https://www.ej-technologies.com/buy/jprofiler/openSource">JProfiler open-source licensing</a></li>
```

Comment:
According to the [open source page](https://www.ej-technologies.com/buy/jprofiler/openSource/enter), the link has to be to https://www.ej-technologies.com/products/jprofiler/overview.html. Could you please update the link? Thanks again for doing this!

Reply:
Opened a PR for this here: https://github.com/foam-framework/foam/pull/569